src/databricks/labs/ucx/hive_metastore/tables.py

```diff
@@ -80,6 +80,15 @@ def sql_unset_upgraded_to(self, catalog):
             f"UNSET TBLPROPERTIES IF EXISTS('upgraded_to');"
         )
 
+    def is_dbfs_root(self) -> bool:
+        if not self.location:
+            return False
+        if self.location.startswith("/dbfs/mnt") or self.location.startswith("dbfs:/mnt"):
+            return False
+        if self.location.startswith("/dbfs/") or self.location.startswith("dbfs:/"):
```

Comment:
what about databricks datasets?

Reply:
Do they show up as managed table?

Reply:
The are located in dbfs:/databricks-datasets and will be handled like any other dbfs root location.